lib/hutch/broker.rb

```diff
@@ -184,7 +184,7 @@ def bindings
 
       filtered = api_client.bindings.
         reject { |b| b['destination'] == b['routing_key'] }.
-        filter { |b| b['source'] == @config[:mq_exchange] && b['vhost'] == @config[:mq_vhost] }
+        select { |b| b['source'] == @config[:mq_exchange] && b['vhost'] == @config[:mq_vhost] }
```

Comment:
How is this change relevant?

Reply:
This fixes build failures for older ruby versions, see travis builds: https://travis-ci.org/gocardless/hutch/jobs/652451121?utm_medium=notification&utm_source=github_status

Reply:
Understood, thanks.